app/api/daily-question/redirect/route.ts

```
@@ -0,0 +1,54 @@
+import { NextRequest, NextResponse } from 'next/server'
+import { createClient } from '@/lib/supabase/server'
+import { getDailyQuestion } from '@/lib/utils/dailyQuestion'
+
+export const dynamic = 'force-dynamic'
+
+/**
+ * API endpoint that redirects to the current daily question
+ * Useful for external sharing or programmatic access
+ */
+export async function GET(request: NextRequest) {
+  let user: any = null
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)